src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldColorpicker.class.php

```diff
@@ -59,11 +59,51 @@ public function GetCMSHtmlFooterIncludes()
     {
         $includes = parent::GetCMSHtmlFooterIncludes();
         $includes[] = '<script src="'.TGlobal::GetStaticURLToWebLib('/javascript/bootstrap-colorpicker-3.0.3/js/bootstrap-colorpicker.min.js').'" type="text/javascript"></script>';
+
         $includes[] = "<script>
-    $(function() {
-        $('#colorPickerContainer".TGlobal::OutHTML($this->name)."').colorpicker({ format: 'hex', useHashPrefix: true, autoInputFallback: false });
-    });
-    </script>";
+            $(function() {
```

Comment:
You could use a "text block" (<<<JS) here?

Reply:
I could, but I don't know if it does us any favors because there are a couple of PHP expressions in there. These would have to become variables which wouldn't make much sense on their own, as they would contain parts of the JavaScript code.

Do you think I should change it nonetheless?

Reply:
I was also wondering about that.

But if there is no easy solution for these expressions I wouldn't do it.